vignettes/susceptibility_matrices.Rmd

```diff
@@ -0,0 +1,390 @@
+---
+title: "Examples of susceptiblity matrices"
+output:
+  bookdown::html_vignette2:
+    fig_caption: yes
+    code_folding: show
+pkgdown:
+  as_is: true
+bibliography: references.bib
+link-citations: true
+vignette: >
+  %\VignetteIndexEntry{Examples of susceptiblity matrices}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  chunk_output_type: console
+---
+
+_finalsize_ requires two matrices as arguments. The matrix format allows to easily represent __multiple__ demographic groups with __multiple__ susceptibility groups.
+
+::: {.alert .alert-warning}
+**New to _finalsize_?** It may help to read the ["Get started"](finalsize.html) vignette first!
+:::
+
+::: {.alert .alert-primary}
+## Use case {-}
+
+You need to create **two susceptibility matrices** to feed two arguments of the `final_size()` function. Here are five case examples with different scenarios for the content of each matrix.
+:::
+
+::: {.alert .alert-secondary}
+### What we have {-}
+
+  1. The susceptibility of a given number of demographic groups per a given number of susceptibility groups;
+  2. The probability of the population in each demographic group and susceptibility group.
+
+### What we assume {-}
+
+  1. The complete partitioning of individuals into different demographic and infection risk groups (susceptibility groups).
+:::
+
+
+## _finalsize_ matrices
+
+### Susceptibility matrix
+
+Contains the _**susceptibility**_ of the population in each demographic group that belongs to each susceptibility group.
+
+::: {.alert .alert-secondary}
+$$
+i \; demographic \; groups \left.\vphantom{\begin{matrix}
+s_{1,1} & s_{1,2} & \cdots & s_{1,j} \\
+s_{2,1} & s_{2,2} & \cdots & s_{2,j} \\
+\vdots  & \vdots  & \ddots & \vdots  \\
+s_{i,1} & s_{i,2} & \cdots & s_{i,j} 
+\end{matrix}}\right\{
+\overbrace{
+\begin{matrix}
+s_{1,1} & s_{1,2} & \cdots & s_{1,j} \\
+s_{2,1} & s_{2,2} & \cdots & s_{2,j} \\
+\vdots  & \vdots  & \ddots & \vdots  \\
+s_{i,1} & s_{i,2} & \cdots & s_{i,j} 
+\end{matrix}}^{\displaystyle j \; susceptibility \; groups}
+\quad
+s: susceptibility
+$$
```

Comment:
Could you explain further or share an example of how to automate this?

Reply:
Essentially I was wondering whethere there was a way to code the printing of this matrix in Mathjax syntax from within R, as done by {texreg} for regression tables - I actually think there isn't a good way to do this, so since this is a one-time use, we can leave it as it is for now.